specification/cognitiveservices/data-plane/WebSearch/v1.0/WebSearch.json

```diff
@@ -0,0 +1,1086 @@
+          "type": "string"
+        },
+        "snippet": {
+          "description": "A snippet of text from the webpage that describes its contents.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "deepLinks": {
+          "description": "A list of links to related content that Bing found in the website that contains this webpage. The Webpage object in this context includes only the name, url, urlPingSuffix, and snippet fields.",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/WebPage"
+          }
+        },
+        "searchTags": {
+          "description": "A list of search tags that the webpage owner specified on the webpage. The API returns only indexed search tags. The name field of the MetaTag object contains the indexed search tag. Search tags begin with search.* (for example, search.assetId). The content field contains the tag's value.",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/WebMetaTag"
+          }
+        }
+      }
+    },
+    "Query": {
+      "description": "Defines a search query.",
+      "type": "object",
+      "required": [
+        "text"
+      ],
+      "properties": {
+        "text": {
+          "description": "The query string. Use this string as the query term in a new search request.",
+          "type": "string"
+        },
+        "displayText": {
+          "description": "The display version of the query term. This version of the query term may contain special characters that highlight the search term found in the query string. The string contains the highlighting characters only if the query enabled hit highlighting",
+          "readOnly": true,
+          "type": "string"
+        },
+        "webSearchUrl": {
+          "description": "The URL that takes the user to the Bing search results page for the query.Only related search results include this field.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "searchLink": {
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "TimeZoneTimeZoneInformation": {
+      "description": "Defines a date and time for a geographical location.",
+      "x-ms-discriminator-value": "TimeZone/TimeZoneInformation",
+      "type": "object",
+      "required": [
+        "location",
+        "time",
+        "utcOffset"
+      ],
+      "properties": {
+        "location": {
+          "description": "The name of the geographical location.For example, County; City; City, State; City, State, Country; or Time Zone.",
+          "type": "string"
+        },
+        "utcOffset": {
+          "description": "The offset from UTC. For example, UTC-7.",
+          "type": "string"
+        }
+      }
+    },
+    "WebWebGrouping": {
+      "x-ms-discriminator-value": "Web/WebGrouping",
+      "discriminator": "_type",
```

Comment:
[Our Linter is tripping on this value](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/286652036#L589) for not being camel cased. Just to double check, was this name deliberately chosen?

Reply:
Yeah. This naming is expected as Schema names are like that only

Reply:
As per my discussion @stefanvalianu , It seems we are blocked to add x-ms-examples.

Reply:
Yea, we can't actually add any examples until this issue is fixed: https://github.com/Azure/oav/issues/142

Ran into this while trying to add some for the entity search spec

Reply:
Rest assured it's tracked on our side :)

Reply:
@stefanvalianu - We are pretty close to fixing the model validation for polymorphic types based on the logic described [here](https://github.com/Azure/oav/issues/142#issuecomment-337403486). If you have time please take a look at the logic in the previously mentioned link. Asking you since you have a better understanding of discriminators based on the email conversations :).

This is the [PR](https://github.com/Azure/oav/pull/171) that implements that logic.

Reply:
looks like this was fixed today; it would be cool if we didn't block on this, I can do this work for entity search and web search all at once, but won't get to it this week

Reply:
Tracked